ext/random/tests/02_engine/mt19937_error.phpt

```diff
@@ -3,12 +3,12 @@ Random: Engine: Mt19937: error pattern
 --FILE--
 <?php
 
-try {
-    new \Random\Engine\Mt19937(1234, 2);
-} catch (\InvalidArgumentException $e) {
-    echo $e->getMessage() . PHP_EOL;
-}
+new \Random\Engine\Mt19937(1234, 2);
```

Comment:
We could stick with the `try-catch` since the backtrace is not particular useful here.

Reply:
After the test failed due to the catch no longer matching, I ran `--bless` and then simplified the test to remove the `catch`.

Is there a strong preference with regard "testing exceptions"? Personally I find the new variant easier to "review", because the PHP logic is simpler.

Reply:
Yeah, the script logic is simpler, but what if the stack backtrace changes in some way? I prefer not to have these backtraces (unless their contents are important), but others may have different preferences.

Reply:
Okay, I've checked the existing tests for `ValueError`s and indeed they catch; changed.